osidb/tests/test_endpoints.py

```diff
@@ -1581,7 +1581,7 @@ def test_flawreference_update(self, auth_client, test_api_uri):
         """
         Test the update of FlawReference records via a REST API PUT request.
         """
-        flaw = FlawFactory()
+        flaw = FlawFactory(embargoed=False)
```

Comment:
The issue here is that when the flaw is embargoed the user needs to have an embargoed access to be able to update it so the issue is with the user and not with the data. I would be for fixing the user ACLs and leaving the test general. You can do it by simply adding `embargo_access` fixture to the test method parameter

https://github.com/RedHatProductSecurity/osidb/blob/master/conftest.py#L167